### ISSUE TITLE
gta3: add asl features from old splitter to new

### ISSUE THREAD

**Issue by iguaan** [zoton2/LiveSplit.Scripts#19]
Collectibles etc, might also work on this when I get home

**Comment by zoton2** [comment]
Main issue I have with the old autosplitter code is that back then there was no autosplitter settings in LiveSplit so it either guessed settings based on your category names or you had to edit the file itself, which I want to not do.

**Comment by iguaan** [comment]
Yeah, my idea is to just implement them as settings, similar to KZ's vc splitter
